core/src/main/scala/akka/kafka/scaladsl/KafkaAsyncConsumer.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2014 - 2016 Softwaremill <http://softwaremill.com>
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.kafka.scaladsl
+
+import java.util
+
+import akka.Done
+import akka.actor.{ActorRef, ActorRefFactory}
+import akka.kafka.ConsumerSettings
+import akka.kafka.internal.KafkaConsumerActor
+import akka.util.Timeout
+import org.apache.kafka.clients.consumer.{ConsumerRebalanceListener, ConsumerRecord, OffsetAndMetadata}
+import org.apache.kafka.common.TopicPartition
+
+import scala.collection.JavaConverters._
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.language.postfixOps
+
+object KafkaAsyncConsumer {
+  def apply[K, V](settings: ConsumerSettings[K, V])(implicit as: ActorRefFactory): KafkaAsyncConsumer[K, V] = new KafkaAsyncConsumer[K, V] {
```

Comment:
I need a way to create an actor which is responsible for kafka connection. Probably I can push creation to user side. This will leads us to expose `KafkaConsumerActor` to a user. What do you think @patriknw, @kciesielski?